public/stmyec_wcm_2022/meta_study.txt

```diff
@@ -0,0 +1,8 @@
+type_of_cancer: stmyec
+cancer_study_identifier: stmyec_wcm_2022
+name: Myoepithelial Carcinomas of the Soft Tissue (CSH Molecular Case Studies, 2022)
+description: Whole Exome Sequencing results (mutation and copy number data) for Weill Cornell Medicine Myoepithelial Carcinomas of the Soft Tissue cohort
```

Comment:
Are these with matched normals?

Reply:
Yes. I'll update in the description as well.

Reply:
@ritikakundra : Description changed to reflect changes